pie/models/scorer.py

```diff
@@ -94,13 +95,20 @@ def get_scores(self):
 
         # compute scores for unknown input tokens
         unk_trues, unk_preds, amb_trues, amb_preds = [], [], [], []
+        unk_targ_trues, unk_targ_preds = [], []
```

Comment:
this is a bit stupid from myself, but could we use `trg` for targets (I think I use it like that in other places): `unk_trg_trues`.

Reply:
Yes